server/src/lib/lazy-cache.ts

```diff
@@ -51,8 +44,7 @@ function redisCache<T, S>(
   timeMs: number
 ): Fn<T, S> {
   return async (...args) => {
-    const { RELEASE_VERSION } = getConfig();
-    const key = cacheKey + JSON.stringify(args) + (RELEASE_VERSION || '');
+    const key = JSON.stringify(args);
```

Comment:
Confusingly the key for in memory cache was different to the Redis one before we made them the same, but we need this to be:

```js
const key = cacheKey + JSON.stringify(args);
```